s3repo/model.py

```diff
@@ -287,6 +346,49 @@ def _get_repository_list(self):
 
         return repos_list
 
+    @staticmethod
+    def _get_abs_path(base_path, path):
```

Comment:
Don't make it `static` and get `base_path` from `self`.

Reply:
Done.